src/components/AssetCard.vue

```diff
@@ -15,7 +15,7 @@
       <object type="image/svg+xml" :data="cardData.preview" class="svgAsset"></object>
     </v-card-media>
     <v-card-media v-else-if="cardData.preview" :src="cardData.preview" height="200px" contain></v-card-media>
-    <v-chip v-if="cardData.fanmade" color="accent" text-color="black">Fanmade Asset</v-chip>
+    <v-chip :style="{visibility: cardData.fanmade ? 'visible' : 'hidden'}" color="accent" text-color="black">Fanmade Asset</v-chip>
```

Comment:
Looks good, could do with extracting it to a class in the same file under `<style>` tags. Since it's visibility it will be the same. Alongside the class that sets visibility to hidden I would suggest this:

```suggestion
    <v-chip :class="{'is-hidden': cardData.fanmade}" color="accent" text-color="black">Fanmade Asset</v-chip>
```